dts/bindings/i2c/ite,enhance-i2c.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2022 ITE Corporation. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+description: ITE it8xxx2 I2C
+
+compatible: "ite,enhance-i2c"
+
+include: i2c-controller.yaml
+
+properties:
+    reg:
+      required: true
+
+    label:
+      required: true
```

Comment:
Label is already required by i2c-controller.yaml.  I don't think it needs to be added here.

Reply:
Removed.